br/pkg/utils/schema_test.go

```diff
@@ -6,39 +6,36 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"os"
+	"testing"
 
 	"github.com/golang/protobuf/proto"
-	. "github.com/pingcap/check"
 	backuppb "github.com/pingcap/kvproto/pkg/brpb"
 	"github.com/pingcap/kvproto/pkg/encryptionpb"
 	"github.com/pingcap/tidb/br/pkg/metautil"
 	"github.com/pingcap/tidb/br/pkg/storage"
 	"github.com/pingcap/tidb/parser/model"
 	"github.com/pingcap/tidb/statistics/handle"
 	"github.com/pingcap/tidb/tablecodec"
+	"github.com/stretchr/testify/require"
 )
 
-type testSchemaSuite struct {
-	store storage.ExternalStorage
-}
-
-var _ = Suite(&testSchemaSuite{})
-
-func (r *testSchemaSuite) SetUpSuite(c *C) {
-	var err error
-	base := c.MkDir()
-	r.store, err = storage.NewLocalStorage(base)
-	c.Assert(err, IsNil)
-}
-
 func mockBackupMeta(mockSchemas []*backuppb.Schema, mockFiles []*backuppb.File) *backuppb.BackupMeta {
 	return &backuppb.BackupMeta{
 		Files:   mockFiles,
 		Schemas: mockSchemas,
 	}
 }
 
-func (r *testSchemaSuite) TestLoadBackupMeta(c *C) {
+func TestLoadBackupMeta(t *testing.T) {
+	t.Parallel()
+
+	testDir, err := os.MkdirTemp("", "load_backup_meta_test")
+	require.NoError(t, err)
+	defer os.RemoveAll(testDir)
```

Comment:
```suggestion
	testDir := t.TempDir()
```

https://pkg.go.dev/testing#T.TempDir